testing/templates/rust-macro-args.html

```diff
@@ -0,0 +1,26 @@
+{{
+        call_a_or_b_on_tail!(
+            (a: compute_len, b: zero),
+                the recursive part that skips over all these
+                tokens doesn't much care whether we will call a
```

Comment:
The single quotes would fail in this case.

Reply:
Good point! I'm okay with that, though, because this would also fail if you put this text in a normal Rust macro call.

Reply:
Ok, this pull request is already finished.